### PR TITLE
[TR] Fix cells selection in grid context

### PR DESCRIPTION
**Pull request by phaseinducer**
This will fix broken scenarios about the proposals grid.

### CODE REVIEW

File: features/Context/Page/Base/Grid.php [hunk 1]
```diff
@@ -776,7 +776,9 @@ public function selectRow($value, $check = true)
      */
     protected function getRowCell($row, $position)
     {
-        $cells = $row->findAll('css', 'td');
+        // $row->findAll('css', 'td') will not work in the case of nested table (like proposals changes)
```
Comment (juliensnz) [inline features/Context/Page/Base/Grid.php]: please remove this line

**Comment by juliensnz** [comment]
GTM

**Comment by solivier** [comment]
GTM

File: features/Context/Page/Base/Grid.php (hunk 1 above)
Comment (jjanvier) [inline features/Context/Page/Base/Grid.php]: I find it useful personnaly

**Pull request merged**